reports/Ohio.md

```diff
@@ -361,6 +361,19 @@ id: oh-columbus-17
 * https://twitter.com/greg_doucette/status/1269118992540749831
 
 
+### Police casually pepperspray passers by | (believed to be) June 1st
+
+Two squad cars full of police arrive at an intersection, sirens wailing. Police get out and start pepper-spraying people. Notably, there are two people on a fire escape who look like they may be treating two others for pepper-spray or tear gas exposure. Police pepper-spray them. Police later pepper spray one a second time, as she's in the process of retreating and rinsing her eyes with milk due to the first pepper spray exposure.
+
+tags: pepper-spray, spray
+
+id: oh-columbus-21
+
+**Links**
+
+* https://twitter.com/aaron_rigg/status/1267649768668495878
+
```

Comment:
https://twitter.com/aaron_rigg/status/1267814008859443202

Add this link too, please